raami/routes.js

```diff
@@ -0,0 +1,38 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET home page. */
+// allow cors for debug reasons
+router.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", process.env.CORS_ORIGIN);
+  res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE,OPTIONS');
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  res.header("Access-Control-Allow-Credentials",'true')
+  next();
+ });
+
+function authenticate(req, res, next) {
+  if (req.session && req.session.user && req.session.user.email) next();
+  else res.status(401).json({ status: 'unauthorized' });
+}
+
+router.use(authenticate);
```

Comment:
This will only verify that the user is signed in; it does not verify whether the user should have any access to this specific artist's information. Also, we should add a `raami_admin` entry [here](/worldcon75/api/blob/master/postgres/init/10-admin-init.sql#L2) and [here](/worldcon75/api/blob/master/kansa/server/lib/user.js#L39) that grants wider access.